server/src/frameworks/Hardhat/worker/WorkerProtocol.ts

```diff
@@ -66,6 +68,26 @@ export class FileBelongsResponse extends ResponseMessage {
   }
 }
 
+export class ResolveImportRequest extends RequestMessage {
```

Comment:
In typescript I think it is more idiomatic to use a type if it is just a data carrier you want.

Reply:
After giving it some thought, you are right that in this case classes are being used incorrectly. These objects are being serialized/deserialized through the ipc between parent and child processes. So they are acting as just plain objects, losing their prototype. I'll change this on the other big PR and on this one